### DISCUSSION TITLE
Customized log

### DISCUSSION THREAD

**Discussion by shinianyue** [gabime/spdlog#2757] [Ideas]
Hi,guys 🖐️ :)!
I tried a little on customized log message.
(In a rather stupid way actually)
I defined a new level "CUSTOMIZED" in the enum.
It's now the highest level (7,higher than "off"level).
Basic functions are almost directly copied from other levels like:
` spdlog::customized("message"); `
And I use a global variable 
`bool customized_should_log_tag` to tell whether it should be logged or not. This variable will be checked in `should_log()`.
As a new learner, your advice will help me a lot.

BTW, I still take it such a tough mission(final assignment)for a freshman (even not CS major) to give this wonderful program some improvements.

**Comment by tt4g** [comment]
What do you want to ask?

**Reply by tt4g** [reply]
Maybe related? #2734

**Comment by shinianyue** [comment]
Thanks for the fast response. Actually I have the same problem as #2703 ,And I just found my answer there and in #1794 .

**Reply by tt4g** [reply]
Yes, custom log levels are not supported in v1.x.
